client/app/expenses/types/expenseType.ts

```diff
@@ -0,0 +1,50 @@
+type User = {
```

Comment:
Oh, in this case, you always only have one file for expenseType, so you can have them in `index.ts`. I did it this way for Tasks. What do you think?

Reply:
I think I might add more types for expense-per-month, so I changed the name for expense-per-date and will seperate the file for expense-per-month
https://github.com/HAK2024/fairy-share/pull/113/commits/7cab587d90f6cc8c152ce15d6c3ce29d7b11b68f